log/sequencer.go

```diff
@@ -66,38 +65,8 @@ var (
 	QuotaIncreaseFactor = 1.1
 )
 
-// stringSet is a set of strings that can be parsed by flag package.
-type stringSet map[string]bool
-
-func (s *stringSet) String() string {
-	keys := make([]string, 0, len(*s))
-	for k, v := range *s {
-		if v {
-			keys = append(keys, k)
-		}
-	}
-	return strings.Join(keys, ",")
-}
-
-func (s *stringSet) Set(value string) error {
-	*s = make(stringSet)
-	for _, id := range strings.Split(value, ",") {
-		(*s)[id] = true
-	}
-	return nil
-}
-
-// The tree IDs for which sequencer does not store ephemeral node hashes.
-// Trillian releases up to v1.4.1 store the ephemeral hashes, which corresponds
-// to this slice being empty. Release v1.4.2 allows disabling this behaviour
-// for individual, or all trees (denoted by the "*" wildcard). The release
-// after v1.4.2 will switch to "*" behaviour unconditionally.
-var idsWithNoEphemeralNodes stringSet
-
 // TODO(pavelkalinnikov): Remove this flag in the next release.
-func init() {
-	flag.Var(&idsWithNoEphemeralNodes, "tree_ids_with_no_ephemeral_nodes", "Comma-separated list of tree IDs for which storing the ephemeral nodes is disabled, or * to disable it for all trees")
-}
+var _ = flag.String("tree_ids_with_no_ephemeral_nodes", "*", "[Deprecated] Comma-separated list of tree IDs for which storing the ephemeral nodes is disabled, or * to disable it for all trees")
```

Comment:
Can we leave some cleanup issue open to track removing this, and x-ref to the issue from here? Having flags defined in non-main files can be a trip hazard, and I'd like to have it tracked to remove this.

Reply:
Done - created an issue and updated the TODO comment.